<launch>
    <group ns="arduino">
        <node pkg="rosserial_python" name="arduino_drive" type="serial_node.py" output="screen">
            <param name="port" value="/dev/ttyACM0"/>
            <param name="baud" value="57600"/>
        </node>
        <!-- <node pkg="rosserial_python" name="arduino_arm" type="serial_node.py">
            <param name="port" value="/dev/ttyACM1"/>
            <param name="baud" value="57600"/>
        </node> -->
    </group>
    <group ns="psi">
        <node pkg="psi" name="drive_manager" type="drive_manager.py" />
        <node pkg="psi" name="navigation" type="navigation.py" output="screen" />
        <node pkg="psi" name="image_analysis" type="image_analysis.py" />
        <node pkg="psi" name="arm_manager" type="arm_manager.py" />
        <node pkg="psi" name="central" type="central.py" output="screen" />
        <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
            <param name="video_device" value="/dev/video0" />
            <param name="image_width" value="640" />
            <param name="image_height" value="480" />
            <param name="framerate" value="10" />
            <param name="pixel_format" value="yuyv" />
            <param name="camera_frame_id" value="usb_cam" />
            <param name="io_method" value="mmap" />
        </node>
    </group>
</launch>